lib/surface/api.ex

```diff
@@ -41,8 +41,6 @@ defmodule Surface.API do
     :context_get
   ]
 
-  @private_opts [:action, :to]
```

Comment:
I haven't found any reference of private_opts like `:action` or `: to` in the codebase.

Reply:
Those are leftovers from the old `context` API. Good catch!